web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -270,6 +277,10 @@ export default {
     ...mapState('girder', {
       selected: (state) => state.currentDandiset,
     }),
+    ...mapState('publish', {
+      mostRecentDandiset: (state) => state.mostRecentDandiset,
```

Comment:
I don't see where this exists in the store, and it doesn't look like it's used anywhere else in this component either. Why is this here?

Reply:
I thought it might be useful, but decided it was too much for this review. I forgot to remove it here, my b